components/pages/CreateSecret.tsx

```diff
@@ -18,102 +19,140 @@ export const CreateSecret = (props: Props) => {
     useCreateSecret(props);
 
   return (
-    <Box>
-      <Container maxWidth="sm">
-        <Box
-          display="flex"
-          flexDirection="column"
-          justifyContent="center"
-          textAlign="center"
-        >
-          <Box mb={20} mt={7.5}>
-            <SendSecurelyLogoWithName />
-          </Box>
-          <Typography mb={1} variant="h2">
-            {t('create.title')}
-          </Typography>
-          <Typography color="textSecondary" mb={5} variant="body2">
-            {t('create.subtitle')}
-          </Typography>
-          <TextField
-            multiline
-            onChange={(event) => setData(event.target.value)}
-            placeholder="Passwords, credentials, API Keys or anything..."
-            rows={4}
-            sx={{ fontFamily: 'Source Code Pro' }}
-          />
+    <>
+      <Head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta
+          content="One time link to share passwords, keys, and other sensitive data."
+          name="description"
+        />
+        <meta
+          content="Send Securely - Send your secret, securely."
+          property="og:title"
+        />
+        <meta
+          content="One time link to share passwords, keys, and other sensitive data."
+          property="og:description"
+        />
+        <meta
+          content="https://cdn.basistheory.com/images/seo/bt-opengraph.png"
+          property="og:image"
+        />
+        <meta
+          content="Send Securely - Send your secret, securely."
+          property="twitter:title"
+        />
+        <meta
+          content="Send your secret, securely. One time link to share passwords, keys, and other sensitive data."
+          property="twitter:description"
+        />
+        <meta
+          content="https://cdn.basistheory.com/images/seo/bt-opengraph.png"
+          property="twitter:image"
+        />
+        <meta property="og:type" content="website" />
+        <meta content="summary_large_image" name="twitter:card" />
+      </Head>
```

Comment:
None of this head information should be in each component - it should be in _document.tsx

Reply:
There were a couple page specific tags